src/main/java/hw3/components/HeaderMenu.java

```diff
@@ -0,0 +1,15 @@
+package hw3.components;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+import static org.openqa.selenium.support.How.CSS;
+
+public class HeaderMenu {
+    @FindBy(how = CSS, using = ".uui-navigation.nav.navbar-nav.m-l8>li")
```

Comment:
How do you plan interact with the current list of the elements?

Reply:
In this testcase - no plan. But as the exercise was to use Page Object Pattern, I decided to describe the whole (or almost) Different Elements page to reuse it later in other tests (if I need to).

Reply:
Deleted class